screens/stores/StoresScreen.js

```diff
@@ -239,7 +245,7 @@ export default class StoresScreen extends React.Component {
             initialSnap={1}
             enabledInnerScrolling
             enabledGestureInteraction
-            snapPoints={['200%', '45%', '10%']}
+            snapPoints={['500%', '45%', '10%']}
```

Comment:
Nit: You might want to add a comment here for clarity - it's unintuitive from reading the code that 500% is a significant number, or that changing it would cause a bug

Reply:
Included comments with explanation - not sure if this borders on verbose but looks alright now.